src/hardware/pcspeaker.cpp

```diff
@@ -340,31 +337,39 @@ static void PCSPEAKER_CallBack(Bitu len)
 				index+=vol_len;
 			} else {
 				/* Check how long it will take to goto new level */
-				float vol_time=fabsf(vol_diff)/SPKR_SPEED;
-				if (vol_time<=vol_len) {
+				constexpr float spkr_speed = SPKR_POSITIVE_VOLTAGE *
+				                             2.0f /
+				                             0.070f; // TODO:
```

Comment:
Leftover TODO? What's this magic number?

Reply:
The rest of the 'TODO' comment was wrapped lower - I've moved it above:

``` c++
				// Check how long it will take to goto new level
				// TODO: describe the basis for these magic numbers and their effects
				constexpr float spkr_speed = SPKR_POSITIVE_VOLTAGE * 2.0f / 0.070f;
```

I also don't understand their basis or their influence :-)

Reply:
Ah, OK; I followed the source and this magin number was introduced as "70" in 322373756 but there's nothing indicating why this value was picked :(